PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,12 +2,12 @@
 
 ### All Submissions:
 
-* [ ] I have read the CONTRIBUTING document.
-* [ ] My code is written in **Python3** and is ending with _.py_
-* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
-* [ ] I have checked that my submission does pass the test on LeetCode.com 
-* [ ] Does your filename follow the naming Conventions?
-* [ ] Have you linked your PR to an Issue?
+- [ ] I have read the CONTRIBUTING document.
+- [ ] My code is written in **Python3** and is ending with _.py_
+- [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
+- [ ] I have checked that my submission does pass the test on LeetCode.com
+- [ ] Does your filename follow the naming Conventions?
+- [ ] Have you linked your PR to an Issue?
```

Comment:
1., 2. and 4. seems good.
You still didn't reset the template to its original state.
```suggestion
* [ ] I have read the CONTRIBUTING document.
* [ ] My code is written in **Python3** and is ending with _.py_
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [ ] I have checked that my submission does pass the test on LeetCode.com 
* [ ] Does your filename follow the naming Conventions?
* [ ] Have you linked your PR to an Issue?
```


Copy&paste the original content or (my recommendation) read about the several possibilities to undo changes in Git:
[Undoing Things](https://git-scm.com/book/en/v2/Git-Basics-Undoing-Things)

Reply:
Sir I have copied and pasted original content from original PULL_REQUEST_TEMPLATE.md file and commited it.